src/shared/util.js

```diff
@@ -18,6 +18,15 @@ import "./compatibility.js";
 const IDENTITY_MATRIX = [1, 0, 0, 1, 0, 0];
 const FONT_IDENTITY_MATRIX = [0.001, 0, 0, 0.001, 0, 0];
 
+const RenderingIntentFlag = {
+  ANY: 0x01,
+  DISPLAY: 0x02,
+  PRINT: 0x04,
+  ANNOTATION_STANDARD: 0x10,
```

Comment:
This one is not actually used.

Reply:
I've been thinking about possible follow-up work, that the new *internal* renderingIntent format would simplify considerably, but let's indeed leave this value out for now.